x-pack/plugins/triggers_actions_ui/public/application/sections/alert_summary_widget/components/alert_summary_widget_full_size.tsx

```diff
@@ -28,6 +28,7 @@ export interface AlertSummaryWidgetFullSizeProps {
   chartProps: ChartProps;
   recoveredAlertCount: number;
   dateFormat?: string;
+  shouldHideCharts?: boolean;
```

Comment:
nit: Personally, I prefer the name `hideChart` (without s) instead of `shouldHideCharts`, then we will have the component without the chart like:

```
<AlertSummaryWidgetFullSize
        activeAlertCount={activeAlertCount}
        activeAlerts={activeAlerts}
        chartProps={chartProps}
        dateFormat={timeRange.dateFormat}
        hideChart
        recoveredAlertCount={recoveredAlertCount}
        shouldHideCharts={shouldHideCharts}
/>
```

Reply:
Good suggestion, thanks! Yeah, I agree it's a single chart that we a hiding - renamed ✔️